src/pricing/index.njk

```diff
@@ -5,10 +5,20 @@ sitemapPriority: 0.90
 meta:
     description: Discover the pricing options for FlowFuse, featuring Starter, Team, and Enterprise plans. Benefit from volume discounts on Team and Enterprise plans. Get started with a 14-day trial period for the Starter plan.
     keywords: FlowFuse pricing, Cost, Plans, Features, Cloud, Self-hosted
+    faq:
```

Comment:
@zackwasli Can you update this list to include the true most frequent questions? E.g. why pricing for self-hosted is the same as Cloud

Reply:
Yes here is my answers from the other day: https://github.com/FlowFuse/website/issues/1959#issuecomment-2073175913

Does FlowFuse offer annual discounts?
-Yes, we offer a 15% discount for annual subscriptions.

Is there a minimum for purchase?
-Yes, 5 instances on Team and 10 instances on Enterprise.

Why isn't there an option for self-hosted Team?
-Due to the configuration and support required for a self-hosted deployment, it is only possible through the Enterprise tier at this time.

Why is self-hosted the same price as cloud?
-Self-hosted environments require more support and configuration than FlowFuse Cloud, which offset any resources saved.

Reply:
I updated the list